clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/ResizeAsgDescription.groovy

```diff
@@ -17,18 +17,16 @@
 package com.netflix.spinnaker.clouddriver.aws.deploy.description
 
 import com.netflix.spinnaker.clouddriver.model.ServerGroup
+import com.netflix.spinnaker.clouddriver.security.resources.ServerGroupNameable
 
-class ResizeAsgDescription extends AbstractAmazonCredentialsDescription {
-  String serverGroupName
-  String region
+class ResizeAsgDescription extends AbstractAmazonCredentialsDescription implements ServerGroupNameable {
   ServerGroup.Capacity capacity
   List<AsgTargetDescription> asgs = []
 
-  @Deprecated
-  String asgName
```

Comment:
Pretty sure these aren't necessary anymore ... we've got a `DescriptionPreProcessor` that takes all these `@Deprecated` values and places them consistently under `asgs`.